### ISSUE TITLE
Can't wrap a string with filters in parens

### ISSUE THREAD

**Issue by fractaledmind** [hopsoft/fmt#4]
```ruby
def test_format_allows_parens
    template = "Hello (%{name}cyan|bold)"
    actual = Fmt(template, name: "World")
    expected = "Hello (\e[36m\e[1mWorld\e[0m)"
    assert_equal expected, actual
  end
```

This fails as you might expect: `Invalid filter! #<Fmt::Filter name="italic)" value="italic)"> (Fmt::Error)`

**Comment by hopsoft** [comment]
This may be addressed in the next release. I'll get a test case up for it. Thanks for the report.

**Comment by hopsoft** [comment]
This use case should be resolved with #3 and supported in release >=0.3.0; though, the syntax is a little different. See the latest [README](https://github.com/hopsoft/fmt) for more details.

```ruby
Fmt("Hello ({{%{name}|>cyan|>bold}})", name: "Stephen")
#=> "Hello (\e[36m\e[1mStephen\e[0m)"
```
